engine/src/main/scala/cromwell/engine/workflow/lifecycle/execution/WorkflowExecutionActor.scala

```diff
@@ -420,7 +420,7 @@ final case class WorkflowExecutionActor(workflowId: WorkflowId,
 
   onTransition {
     case _ -> toState if toState.terminal =>
-      workflowLogger.info(s"$tag done. Shutting down.")
+      workflowLogger.debug(s"$tag done. Shutting down WorkflowExecutionActor.")
```

Comment:
Right, but the message was still scary... "why is the WorkflowExecutionActor shutting down Cromwell!?!"

Maybe there's a middle ground? `shutting down this actor`?